src/content/docs/dropins/user-auth/slots.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: User auth slots
+description: Learn about the slots provided in the User Auth drop-in component.
+---
+
+This topic describes the slots that are provided in the order drop-in component.
+
+[Extending drop-in components](/dropins/all/extending/) describes the default properties available to all slots.
+
+## FormInput_*attributeCode* and SignUpFormInput_*attributeCode* slots
+
+The `FormInput_*attributeCode*` and `SignUpFormInput_*attributeCode*` slots allow you to customize the input fields in the `Form` and `SignUpForm` forms. The value of `*attributeCode*` is a valid attribute code used in the form. For example, a slot named `SignUpFormInput_lastname` would customize the `lastname` input field in the `SignUpForm` form.
+
+The following example illustrates the usage of the `SignUpFormInput_lastname` slot:
```

Comment:
I apologize for any confusion caused. I’m not entirely sure where the root of the misunderstanding lies, but I suspect there might have been an error in my draft documentation.

To clarify, the auth drop-in has only one slot of this type: `SignUpFormInput_*attributeCode*`. There is no `FormInput_*attributeCode*`.

Aside from that, the rest of the documentation and code examples are correct. Thank you for bringing this to my attention!